app/controllers/mixins/image_validation_mixin.rb

```diff
@@ -0,0 +1,34 @@
+module Mixins
+  module ImageValidationMixin
+    extend ActiveSupport::Concern
+ 
+    private
+
+    # @param file request parameter for a file
+    # @param 
+    def valid_image_file?(file, type = nil)
+      ext = file.original_filename.split(".").last.downcase
```

Comment:
Alternatively, `File.extname`

Reply:
`File.extname` keeps the `"."`.
When I changed the code over to that it ended up changing too much.

Deciding to keep it the way it was when I came in